src/presentation/pages/Note.vue

```diff
@@ -4,11 +4,9 @@
       :style="{ '--opacity': id && note ? 1 : 0 }"
     >
       <template #left>
-        {{
-          note && 'updatedAt' in note && note.updatedAt
-            ? t('note.lastEdit') + ' ' + getTimeFromNow(note.updatedAt)
-            : t('note.lastEdit') + ' ' + 'a few seconds ago'
-        }}
```

Comment:
you should not remove "Last edit" info. It should be aligned to the right side:

<img width="1058" alt="image" src="https://github.com/user-attachments/assets/75aaf414-75ff-4ec8-872b-611405799ef9" />